src/devicecontroller/NoOpDeviceController.ts

```diff
@@ -69,6 +70,23 @@ export default class NoOpDeviceController
   getVideoInputQualitySettings(): VideoQualitySettings | null {
     return null;
   }
+  getEventController(): EventController {
```

Comment:
Could you please let me know why we need ` getEventController` and `setEventController` when we are adding the `eventController` dependency through the constructor? Also, the `eventController` is not private in the `NoOpDeviceControllerWithEventController` class.

Reply:
It was unintentional that the `eventController` was a required dependency in the constructor and that it was public, so I updated it. The main purpose of the `NoOpDeviceControllerWithEventController` as well as the getter and setters is to test to make sure `MeetingSession` does not overwrite the `EventController` in the `DeviceController` and that it will set one if there isn't already one.